src/lib/coda_base/snark_transition.ml

```diff
@@ -120,13 +137,15 @@ module Make (Inputs : Inputs_intf) :
       ; consensus_data
       ; sok_digest
       ; supply_increase
+      ; prover_state
```

Comment:
Should prover_state really just be passed through in the typ like this? Shouldn't prover_state define it's own typ?

Reply:
Actually I'm going to start the restructuring now as the current way is just so messy

Reply:
ok, done